### ISSUE TITLE
Option to display 240i/240p content in 480i/480p

### ISSUE THREAD

**Issue by cheatfreak47** [FIX94/Nintendont#274]
Something I've noticed is some newer HD TVs don't support 240i/240p these days, and Nintendont seems like it'd be capable of fixing this, I myself have one of these HDTVs, and it's rather annoying that I can't play any game that uses the 240 display modes.

While I have no idea how feasible it is to actually scale the video up to 480, it'd be a nice feature for those that need it like myself.

**Comment by ppsspp213** [comment]
It's not just 240i/p it's 288i/p also, Samsung led hdtv oddly support 240p but not 288i/p.

**Comment by GB-CG** [comment]
It depends, if one uses a Wii-to-HDMI adapter-
![wiitohdmi](https://cloud.githubusercontent.com/assets/11931125/22405203/deb363e6-e636-11e6-92e8-f4c368dcadaa.jpg)
-it certainly won't pick up 240p/288p. But with the Component cables, it's viewed as 480i on some TVs like the Panasonic Viera's.

**Comment by cheatfreak47** [comment]
Right, but many TV's just don't support the signal type outright, and it's somewhat common for Gamecube games, compared to Wii games, so, the suggestion is to scale it up to 480p/i in software or something.

**Comment by carnage702** [comment]
nintendont doesnt emulate any graphics generated by the gc thats why its a seminative app, it runs cpu and gx stuff all native, we have no way to upscale a videomode becuase we are not emulating videomodes or graphics at all.

**Comment by cheatfreak47** [comment]
Hmm, well, perhaps it's possible to just forcibly make the game render in 480i or 480p and no upscaling would be necessary.
